bp/bp.xcodeproj/project.pbxproj

```diff
@@ -1017,6 +1019,7 @@
 			buildSettings = {
 				DEPLOYMENT_LOCATION = NO;
 				"DEVELOPER_PRIVATE_FRAMEWORKS_DIR[arch=*]" = /Library/Developer/PrivateFrameworks/;
+				DSTROOT = "";
```

Comment:
all the dstroot and install_root changes are probably not needed.

Reply:
This is needed for Xcode 15